packages/hooks/src/__tests__/useTotalAssetCurrency.spec.tsx

```diff
@@ -0,0 +1,242 @@
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useRealTotalAssetCurrency(), { wrapper });
+        expect(result.current).toBe('EUR');
+    });
+
+    test('should return default currency when user has no real account', async () => {
+        const mock = mockStore({
+            client: {
+                default_currency: 'EUR',
+                accounts: {
+                    acc1: {
+                        currency: 'JPY',
+                        is_virtual: 1,
+                    },
+                },
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useRealTotalAssetCurrency(), { wrapper });
+        expect(result.current).toBe('EUR');
+    });
+
+    test('should return proper currency when user has non_crypto account', async () => {
+        const mock = mockStore({
+            client: {
+                is_crypto: (currency: string) => currency === 'BTC',
+                accounts: {
+                    crypto_acc: {
+                        currency: 'BTC',
+                        is_virtual: 0,
+                    },
+                    non_crypto_acc: {
+                        currency: 'AUD',
+                        is_virtual: 0,
+                    },
+                },
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useRealTotalAssetCurrency(), { wrapper });
+
+        expect(result.current).toBe('AUD');
+    });
+
+    test('should return empty string when user has non_crypto account with no currency set', async () => {
+        const mock = mockStore({
+            client: {
+                is_crypto: (currency: string) => currency === 'BTC',
+                accounts: {
+                    crypto_acc: {
+                        currency: 'BTC',
+                        is_virtual: 0,
+                    },
+                    non_crypto_acc: {
+                        is_virtual: 0,
+                    },
+                },
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        const { result } = renderHook(() => useRealTotalAssetCurrency(), { wrapper });
+
+        expect(result.current).toBe('');
+    });
+
+    test("should return the first account's currency when user only has crypto account", async () => {
+        const mock = mockStore({
+            traders_hub: {
+                is_eu_user: false,
+            },
+            client: {
+                is_crypto: (currency: string) => ['BTC', 'ETH'].includes(currency) || true,
```

Comment:
```suggestion
                is_crypto: (currency: string) => ['BTC', 'ETH'].includes(currency),
```

Reply:
since we are calling `is_crypto()` without passing any args in the hook, I need to have the `|| true` in the mocked funciton